storage/blob.go

```diff
@@ -590,6 +602,8 @@ func (b BlobStorageClient) GetBlobProperties(container, name string) (*BlobPrope
 		ContentLength:         contentLength,
 		ContentEncoding:       resp.headers.Get("Content-Encoding"),
 		ContentType:           resp.headers.Get("Content-Type"),
+		CacheControl:          resp.headers.Get("Cache-Control"),
```

Comment:
@schreibe72 thanks for making the changes. I think we should somehow embed BlobHeaders struct into BlobProperties struct and have some deserializer that does map-->struct. I feel like it might be a lot of work whereas this patch is probably solving your problem. What do you think?